ajax/libs/moonjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "moonjs",
+  "filename": "moon.min.js",
+  "version": "0.10.0",
+  "description": "A minimal, blazing fast UI library.",
+  "homepage": "http://moonjs.ga/",
+  "keywords": [
+    "moon",
+    "mvvm",
+    "user interface",
+    "UI"
+  ],
+  "author": "Kabir Shah",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KingPixil/moon"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/KingPixil/moon",
```

Comment:
Let's change to `"target": "git://github.com/KingPixil/moon.git",`
This can let git auto-update work (to download the files), otherwise, it can't download the file.